sql/moz-fx-data-shared-prod/telemetry_derived/cohort_daily_statistics_v2/query.sql

```diff
@@ -0,0 +1,150 @@
+-- Get all clients who had activity in the last 28 days and their bit patterns.
+WITH submission_date_activity AS (
+  SELECT
+    client_id,
+    days_seen_bits,
+    submission_date as activity_date
+  FROM
+    telemetry.clients_last_seen_v1 -- this might cause an issue because definition of first_seen_date in this table is different from clients_first_seen_v2
+  WHERE
+    submission_date = @activity_date
+  GROUP BY
+    client_id,
+    submission_date,
+    days_seen_bits
+),
+-- Get all the cohorts that are still in range of the current day of activity (180 days)
+cohorts_in_range AS (
+  SELECT
+    client_id,
+    first_seen_date AS cohort_date,
+    DATE(@activity_date) AS activity_date,
+    -- activity_segment, -- for desktop: from clients_last_seen, for mobile: calculated field from mobile_with_searches in unified_metrics
+    app_version,
+    attribution_campaign,
+    attribution_content,
+    attribution_experiment,
+    attribution_medium,
+    attribution_source,
+    attribution_ua,
+    -- attribution_variation, -- can find in clients_daily_v6 with unpacking attribution field
+    partner_distribution_version,
+    partner_distributor,
+    partner_distributor_channel,
+    partner_id,
+    city,
+    country,
+    -- device_model, -- important for mobile, usually null, may not need
+    apple_model_id, -- from clients_first_seen_v2
+    distribution_id,
+    -- is_default_browser,  -- can find in clients_daily_v6
+    locale,
+    -- normalized_app_name,  -- strings that come from unified_metrics/filters out BrowserStack
+    app_name as normalized_app_name, -- from clients_first_seen_v2, as per Lucia's msg app_name = normalized_app_name
+    normalized_channel,
+    normalized_os,
+    normalized_os_version,
+    -- os_version_major,
+    -- os_version_minor,
+    COALESCE(
+      SAFE_CAST(NULLIF(SPLIT(normalized_os_version, ".")[SAFE_OFFSET(0)], "") AS INTEGER),
+      0
+    ) AS os_version_major,
+    COALESCE(
+      SAFE_CAST(NULLIF(SPLIT(normalized_os_version, ".")[SAFE_OFFSET(1)], "") AS INTEGER),
+      0
+    ) AS os_version_minor,
+  FROM
+    telemetry_derived.clients_first_seen_v2
+  WHERE
+    first_seen_date
+    BETWEEN DATE_SUB(@activity_date, INTERVAL 180 DAY)
```

Comment:
One awkward thing about 180 days is that its not equally divisible by 28, which is one of our main retention intervals. 196 would give us 7 complete months, and would tell us if any clients make it into their second half their first year.

Reply:
Sounds reasonable to me. Will make this change